services/src/main/java/org/fao/geonet/api/standards/StandardsApi.java

```diff
@@ -23,46 +23,36 @@
 
 package org.fao.geonet.api.standards;
 
-import com.itextpdf.text.Meta;
 import io.swagger.annotations.*;
+import jeeves.server.context.ServiceContext;
```

Comment:
A lot of changes to import order - does geonetwork have a standard that we can configure IDE to follow?